packages/datadog-plugin-avsc/test/schemas/user.avsc

```diff
@@ -0,0 +1,9 @@
+{"namespace": "example.avro",
```

Comment:
This JSON files seem a bit chaotic which might make future changes to them messy in git. Can you clean them up into a consistent format?

```sh
python -m json.tool user.avsc
```

Reply:
Ive separated out the expected schemas json into separate files. Also applied formatting to all json files. Should be all set.